BWT/BWT/Tests.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace BWT
+{
```

Comment:
Используйте [file-scoped namespaces](http://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces)
```suggestion
﻿namespace BWT;
```